### ISSUE TITLE
Warning: NLS missing message: GotoMatchingBracket_error_noMatchingBracket in: org.eclipse.ui.internal.genericeditor.messages

### ISSUE THREAD

**Issue by vogella** [eclipse-platform/eclipse.platform#857]
Opening the target editor source tab I see the following message on the console:

Warning: NLS missing message: GotoMatchingBracket_error_noMatchingBracket in: org.eclipse.ui.internal.genericeditor.messages

Screenshot of the UI I'm opening:

![image](https://github.com/eclipse-platform/eclipse.platform/assets/139910/bf60a7f1-7ee3-460e-a389-3e701109560f)

**Comment by merks** [comment]
Yes, it's been that way for a while and is not the only missing thing:

![image](https://github.com/eclipse-platform/eclipse.platform/assets/208716/8b3e266d-6e74-4510-bac2-487692067ab8)

Probably a file missed in the commit or a copy and paste error.  JDT defines them like this:
```
GotoMatchingBracket_error_noMatchingBracket=No matching bracket found
GotoMatchingBracket_error_bracketOutsideSelectedElement=Matching bracket is outside the selected element
```
